sbpy/data/ephem.py

```diff
@@ -127,35 +129,137 @@ def from_horizons(cls, targetids, id_type='smallbody',
         return cls.from_table(all_eph)
 
     @classmethod
-    def from_mpc(cls, targetid, epoch, observatory='500'):
-        """
-        Load ephemerides from the
+    def from_mpc(cls, targetid, epochs=None, start=None, step=None,
```

Comment:
how do you feel about allowing a dictionary with keywords `start`, `stop`, and `step` for `epochs` instead of providing these parameters on the top level? That would be the same approach I've been taking in `from_horizons` and I would like to keep the interfaces as similar as possible

Reply:
Done.